Getting Started section rehaul (#28090)
grafana/grafana: docs/sources/getting-started/_index.md

[menu.docs]
name = "Getting started"
identifier = "getting-started"
weight = 10
+++

# Getting started

> These section will help first time users get acquainted with Grafana. Grafana is open source visualization and analytics software. To learn more, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).

Reviewer: ```suggestion
These sections help first-time users to get acquainted with Grafana. Grafana is an open-source platform for monitoring and observability. To learn more, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
```
